### PR TITLE
[BACK-1015] Add new DeviceTransmitterEvent

### PR DESCRIPTION
**Pull request by jamesraby**
Add the new DeviceTransmitterEvent to data model.

Change name of DeviceSensor to Sensor

### CODE REVIEW

File: data/types/device/sensor/devicesensor.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package devicesensor
+package sensor
```
Comment (pazaan) [inline data/types/device/sensor/devicesensor.go]: Suggest that this file be renamed to `sensor.go` for consistency.

File: data/types/device/sensor/devicesensor_suite_test.go [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package devicesensor_test
+package sensor_test
```
Comment (pazaan) [inline data/types/device/sensor/devicesensor_suite_test.go]: Suggest that this file be renamed to `sensor_suite_test.go` for consistency.

File: data/types/device/sensor/devicesensor_test.go [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package devicesensor_test
+package sensor_test
```
Comment (pazaan) [inline data/types/device/sensor/devicesensor_test.go]: Suggest that this file be renamed to `sensor_test.go` for consistency.

File: data/types/device/sensor/devicesensor.go (hunk 1 above)
Comment (jamesraby) [reply data/types/device/sensor/devicesensor.go]: agreed

File: data/types/device/sensor/devicesensor_test.go (hunk 3 above)
Comment (jamesraby) [reply data/types/device/sensor/devicesensor_test.go]: done

File: data/types/device/sensor/devicesensor_suite_test.go (hunk 2 above)
Comment (jamesraby) [reply data/types/device/sensor/devicesensor_suite_test.go]: done

**Pull request merged**